package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "blood-connect",
+  "description": "Blood donation management application",
+  "author": "Craftsmen Ltd.",
+  "version": "1.0.0",
```

Comment:
Can we start with 0.0.1 please?

Reply:
modified in commit [1f42939](https://github.com/CraftsmenLtd/BloodConnect/pull/44/commits/1f429390b0c796bedc7aa46bed06dfa227cd3adb)

Reply:
@melon-ruet bhai can you please confirm that this has been resolved, and if so, approve the PR?